frontend/src/components/cluster-dashboard/ClusterDashboard.tsx

```diff
@@ -1,17 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect, Dispatch, SetStateAction } from 'react';
+import { Grid } from '@material-ui/core';
+import axios from 'axios';
+
+import { akkaClusterProps } from './../../assets/properties/akkaClusterProps';
 import './ClusterDashboard.scss';
-import { Grid, Paper } from '@material-ui/core';
+import ClusterListView from './cluster-list-view/ClusterListView';
+import { Cluster } from './Cluster.model';
+import SimpleSnackBar from '../shared/simple-snack-bar/SimpleSnackBar';
 
 const ClusterDashboard: React.FC = () => {
+  
+  const [cluster, setCluster]: [Cluster, Dispatch<SetStateAction<Cluster>>] = useState(new Cluster());
+  const [openSnackBar, setOpenSnackBar] = React.useState(false);
+  const [snackBarMessage, setSnackBarMessage] = React.useState("");
+  
+  const loadClusterData = (): void => {
+    const akkaGetClusterMembersUrl = akkaClusterProps["akka.management.url"] + "/cluster/members";
+    const getClusterResponse = (): void => {
+      axios.get(akkaGetClusterMembersUrl)
+        .then(response => {
+          if(response.status === 200) { 
+            const clusterData: Cluster = response.data;
+            setCluster(clusterData)
+          }
+        })
+        .catch(function (error) {
+          console.log(error);
+          setSnackBarMessage(error.message && error.message.message ? error.message.message : error.message);
+          setOpenSnackBar(true);
+        })
+    };
+    getClusterResponse();
+  }
+
+  useEffect(loadClusterData, []);
+
   return (
     <React.Fragment>
       <Grid container md item direction="row" className="shukra-body">
-        <Grid item xs={3}>
-          <Paper />
-        </Grid>
-        <Grid item xs={9}>
-          <Paper />
+        <ClusterListView clusterData={cluster} refreshClusterData={loadClusterData}/>
+        <Grid item xs={9} className="shukra-right-container">
         </Grid>
+        <SimpleSnackBar message={snackBarMessage} open={openSnackBar} setOpen={setOpenSnackBar}/>
```

Comment:
It seems to me that `SimpleSnackBar` should either completely manage its own state of being open or not, or it should only take a single prop dictating if it is open. This is in part escaping from the idea of unidirectional data flow by allowing the child to explicitly dictate the state of the parent.

Reply:
Changed it to take a single prop dictating if it's open/closed.